softdevice_controller/include/sdc_hci_cmd_controller_baseband.h

```diff
@@ -0,0 +1,587 @@
+/*
+ * Copyright (c) Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file sdc_hci_cmd_controller_baseband.h
+ *
+ * @defgroup sdc_hci_cb SoftDevice Controller HCI Interface for Controller & Baseband commands
+ * @ingroup sdc_hci
+ *
+ * All APIs in this header file are expected to be called from the
+ * same execution priority as mpsl_low_priority_process.
+ * Not doing so will lead to undefined behavior.
+ *
+ * @{
+ */
+
+#ifndef SDC_HCI_CMD_CONTROLLER_BASEBAND_H__
+#define SDC_HCI_CMD_CONTROLLER_BASEBAND_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+#include <cmsis_compiler.h>
```

Comment:
This header is not part of this repository, right. I think that has to be fixed for the stand-alone requirement of these libraries.

Reply:
The MDK (nrf.h) has a dependency to cmsis. Therefore I believe it is ok to have this dependency. If I remember correctly, this was brought up earlier as well.

Reply:
Yes, that is true.